pkgs/by-name/be/beidconnect/package.nix

```diff
@@ -0,0 +1,77 @@
+# Locations:
+#
+# Linux (system-wide)
+#     Google Chrome: /etc/opt/chrome/native-messaging-hosts
+#     Chromium: /etc/chromium/native-messaging-hosts
+#     Edge: /etc/opt/edge/native-messaging-hosts
+#     Firefox: /usr/{lib,lib64,share}/mozilla/native-messaging-hosts
+# Linux (user-specific, default path)
+#     Google Chrome: ~/.config/google-chrome/NativeMessagingHosts
+#     Chromium: ~/.config/chromium/NativeMessagingHosts
+#     Edge: ~/.config/microsoft-edge/NativeMessagingHosts
+#     Firefox: ~/.mozilla/native-messaging-hosts
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gnumake,
+  pcsclite,
+  boost,
+  pkg-config,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "beidconnect";
+  version = "2.10";
+
+  src = fetchFromGitHub {
+    owner = "Fedict";
+    repo = "fts-beidconnect";
+    rev = finalAttrs.version;
+    hash = "sha256-xkBldXOlgLMgrvzm7ajXzJ92mpXrxHD1RX4DeBxU3kk=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    gnumake
+  ];
+
+  buildInputs = [
+    pcsclite.dev
+    boost
+  ];
+
+  outputs = [ "out" ];
+  strictDeps = true;
+
+  buildPhase = ''
+    make -j $(nproc) -C linux CFLAGS='-g -O2' CXXFLAGS='-g -O2'
+  '';
+
+  installPhase = ''
+    install -d $out/bin/ \
+      $out/native-messaging-hosts/chromium/ \
+      $out/native-messaging-hosts/mozilla/ \
+
+    install ./linux/beidconnect $out/bin
+    $out/bin/beidconnect -setup \
+      $out/bin \
+      $out/native-messaging-hosts/chromium \
+      $out/native-messaging-hosts/mozilla
+  '';
+
+  meta = with lib; {
```

Comment:
We’re trying to avoid this because it has some awkward footguns (https://github.com/NixOS/nixpkgs/issues/208242). You could switch the `license` to `lib.licenses.mit` and drop the `with`.